package.json

```diff
@@ -79,6 +79,7 @@
         "react-router": "^5.0.1",
         "react-router-dom": "^5.0.1",
         "request": "^2.88.0",
```

Comment:
Kan vel fjerne avhengigheten til `request`?

Reply:
`request is defined as a peer-dependency and thus has to be installed separately.`